src/components/Map/Map.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect } from 'react';
+import { MapContainer, TileLayer, useMap, Marker, Popup } from 'react-leaflet';
+import farmersMarkets from '../../marketData';
+import { useNavigate } from 'react-router';
+import { useState } from 'react';
+import * as L from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+import markerIcon2x from 'leaflet/dist/images/marker-icon-2x.png';
+import markerIcon from 'leaflet/dist/images/marker-icon.png';
+import markerShadow from 'leaflet/dist/images/marker-shadow.png';
+import { getMarkets } from '../../apiCalls';
+
+//currently our react-typescript version is not showing the default marker.  Chat-gpt suggested creating a custom one
+const customMarkerIcon: L.Icon = L.icon({
+  iconUrl: markerIcon,
+  iconRetinaUrl: markerIcon2x,
+  shadowUrl: markerShadow,
+  iconSize: [25, 41],
+  iconAnchor: [12, 41],
+  popupAnchor: [1, -34],
+  shadowSize: [41, 41],
+});
+
+//types needed for our state object
+type MarketProps = {
+  market_name: string;
+  address: string;
+  lat: string;
+  lon: string;
+};
+
+type selectedMarketProps = [
+  {
+    market_name: string;
+    address: string;
+    lat: string;
+    lon: string;
+    website: string;
+    zipcode: string;
+    phone: string;
+  }
+];
+
+//The newest version of leaflet uses a new hook called useMap
```

Comment:
Ok, gotcha.  We're working with the new version of leaflet here that is using useMap and we need to incorporate the ConfigureMap component to re-introduce the functionality we lost:  center, scrollWheelZoom, and zoom.  Great work in researching that and getting the solution!